src/makeRobot.js

```diff
@@ -38,7 +38,71 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(distance = 1) {
+      if (distance < 0) {
+        return this;
+      }
+
+      this.coords.y += distance;
+
+      return this;
```

Comment:
Так не будеш повторюватися
```suggestion
    goForward(distance = 1) {
      if (distance > 0) {
        this.coords.y += distance;
      }

      return this;
```